tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -281,7 +345,8 @@ def _run_benchmarks(
             "DirectoryType": directory_type,
         }
         metric_publishing_timestamp = datetime.datetime.now()
-        for num_nodes in [100, 250, 500, 1000, 2000]:
+        for num_nodes in [100, 250]:
```

Comment:
minor: we should probably use much smaller compute node counts once these tests are running every day, but this is fine for now.